serverless/integrations/gpu-resources.adoc

```diff
@@ -0,0 +1,39 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="gpu-resources"]
+= Using NVIDIA GPU resources with serverless applications
+:context: gpu-resources
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+NVIDIA support experimental use of GPU resources on {product-title}.
+See link:https://docs.nvidia.com/datacenter/kubernetes/openshift-on-gpu-install-guide/index.html[{product-title} on NVIDIA GPU accelerated clusters] for more information about setting up GPU resources on {product-title}.
+
+Once GPU resources are enabled for your {product-title} cluster, you can specify GPU requirements for a Knative service using the `kn` CLI.
```

Comment:
s/Once/After/
**once** _adv_
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”
p.s. I'm not a fan of this rule...I think "once" has such common usage that it sounds more natural. But ISG 🤷‍♂️

Reply:
lol yep, I keep forgetting about this one, sorry :slightly_smiling_face: thanks!